docusaurus/docs/Android/04-ui/02-general-customization/02-chatui.mdx

```diff
@@ -734,3 +734,220 @@ Which makes the UI look like so:
 Only the video thumbnails fetched from Stream's APIs are paid and can be disabled. Video thumbnails loaded from local storage - such as the ones in the attachment picker - are not paid and will remain enabled.
 :::
 
+## Customizing Message Decorators
+
+Message decorators are used to decorate the message UI elements. 
+For example, the `ReplyDecorator` is used to reflect `Message.replyTo` property on UI.
+
+You can customize the message decorators by overriding the default `ChatUI.decoratorProviderFactory`:
+
+### Removing Built-in Decorators
+
+For instance you can remove one the built-in decorators.
+Let's try to remove the `ReplyDecorator`:
+
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+ChatUI.decoratorProviderFactory = DecoratorProviderFactory.default {
```

Comment:
Could we keep those code-snippets into the docs artifact?
It would avoid issues in our docs 
```suggestion
ChatUI.decoratorProviderFactory = DecoratorProviderFactory.defaultFactory {
```